buildpipeline/pipelines.json

```diff
@@ -24,6 +24,19 @@
           }
         },
         {
+          "Name": "DotNet-CoreClr-Trusted-Linux",
+          "Parameters": {
+            "DockerTag": "centos-6-783abde-20171304101322",
+            "Rid": "rhel6"
```

Comment:
Thanks @weshaggard , changed it to rhel.6.

There is historical reason that in coreclr, the build and test parses RID in different ways. The RID you pass to the test build is the full RID like in dir.props, and in the product build it’s constructed from the OS portion & the arch portion.

This is a product build entry, so should be rhel.6 without -64.